views/default/elements/navigation.css.php

```diff
@@ -485,6 +485,9 @@
 	border-radius: 0 3px 3px 3px;
 	box-shadow: 1px 3px 5px rgba(0, 0, 0, 0.25);
 }
```

Comment:
We just .elgg-menu-hover too generically. I'd rather put these props on .elgg-popup and define more specific selectors for our dropdown menus, user card, etc.